com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -20,6 +20,12 @@ publisher.default = unnamed
 developer.type = string
 developer.help = The developer name
 developer.default = unnamed
+custom_resources.help = A comma separated list of resources that will be included in the project. If directories are specified, all files and directories in that directory are recursively included
```

Comment:
These were missing. Found these when exploring a game.project option for excluding folders (abandoned in favour of current solution).